src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java

```diff
@@ -132,7 +132,7 @@ private Organisation saveOrganisation(Organisation organisation) {
 
     public void addPbaAccountToOrganisation(Set<String> paymentAccounts, Organisation organisation) {
         if (paymentAccounts != null) {
-            paymentAccountValidator.checkPbaNumberIsValid(paymentAccounts);
+            PaymentAccountValidator.checkPbaNumberIsValid(paymentAccounts);
```

Comment:
if you are making call to static method static way, then remove instance variable "paymentAccountValidator" if it is not required anywhere.